### PR TITLE
Three Generations of AWS blog post and style tweaks

### PR DESCRIPTION
**Pull request by nzoschke**
screen shot 2016-09-20 at 9 45 58 am

## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by nzoschke** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;XZZAKelg&#x2F;731-new-blog-post"> New Blog Post</a>

File: _posts/2016-09-20-three-generations-of-aws.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+EC2 Instances are extremely flexible, so numerous strategies for faster deployment times exist. A common technique is to use a tool like Ansible to SSH into every instance and pull a new version of code then restart the app server. But now we’re using bespoke scripts to mutate instances which add to the failure scenarios.
+
+A cutting-edge technique is to install our own container orchestration software like Docker Swarm or Apache Mesos to coordinate starting and stopping processes across a cluster of EC2 instances. Big companies like Google and Twitter have proven this model of scheduling work across a cluster of generic Instances.
+
+EC2 Instances with container orchestration **is** the best practice. 
+
+This explains why there is heated competition in the orchestration software space, and why AWS launched their EC2 Container Service (ECS).
+
+If we install Swarm or Mesos we're responsible for operating the software. We need to keep an orchestration service up 100% of the time so Instances can always check in and see if they need to start or stop processes. If we use ECS, we delegate that responsibility to Amazon.
```
Comment (mattmanning) [inline _posts/2016-09-20-three-generations-of-aws.md]: Needs a comma after "time".

File: _posts/2016-09-20-three-generations-of-aws.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+Amazon provides an “ECS Optimized AMI” that has the right server operating system, Docker server pre-configured. AWS is responsible for keeping the ECS APIs up and running so instances can always connect to it to ask for more work. AWS is responsible for writing the open-source ecs-agent that runs on every instance.
+
+Because AWS built the entire stack, we can trust its quality, and trust that AWS will support it through tickets if things don't work right.
+
+It’s also important to understand that AWS considers Tasks a first-class primitive of the entire AWS platform. Tasks can be configured to assume an IAM role, register with a load balancer, and to log to a CloudWatch Log Group.
+
+This year we saw two new services, Elastic File System (EFS) and Application Load Balancer (ALB) which complement ECS very well.
+
+In many ways Tasks have more platform more features out of the box than EC2 Instances ever did.
```
Comment (mattmanning) [inline _posts/2016-09-20-three-generations-of-aws.md]: I think "more platform more features" here is supposed to be "more platform features"?

**Comment by nzoschke** [comment]
@mattmanning I just updated based on your feedback. Thanks!

**Comment by ddollar** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;RLUcjbK8&#x2F;749-three-generations-blog"> three generations blog</a>

**Pull request merged**